src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/NavigationTests.java

```diff
@@ -76,4 +76,26 @@ public void MercuryCuratedNavigationTests_002_navigateThroughSection() {
 
     UrlChecker.isPathContainedInCurrentUrl(driver, ROOT_PATH_SECTION + section.getTitle());
   }
+
+  // CCT09
+  @Test(groups = {"MercuryCuratedNavigationTests_003", "MercuryCuratedNavigationTests",
+                  "MercuryCuratedContentTests", "Mercury"})
+  public void MercuryCuratedNavigationTests_003_navigateThroughDifferentUrl() {
+    CuratedContentPageObject section = new CuratedContentPageObject(driver);
+
+    String expectedUrl = wikiURL + MercurySubpages.CC_CATEGORY_TEMPLATES;
+    section.navigateToUrlWithPath(wikiURL, MercurySubpages.CC_CATEGORY_TEMPLATES);
+    UrlChecker.isUrlEqualToCurrentUrl(driver, expectedUrl);
+
+    expectedUrl = wikiURL + MercurySubpages.CC_SECTION_CATEGORIES;
+    section.navigateToUrlWithPath(wikiURL, MercurySubpages.CC_SECTION_CATEGORIES);
+    UrlChecker.isUrlEqualToCurrentUrl(driver, expectedUrl);
```

Comment:
this test won't work on different environments than prod. Because if you add sandbox-X to URL then currentURL won't match expected one